west.yml

```diff
@@ -62,7 +62,7 @@ manifest:
       revision: 6e3316da412649f02d133231078427e452f65db6
       path: modules/lib/gui/lvgl
     - name: mbedtls
-      revision: ca32746072ce3381f1c9ae46ba6cd34c69f8c0ee
+      revision: pull/2/head
```

Comment:
Should this be a tag instead of a pull branch on github?

Reply:
@d3zd3z Until the fixes are merged in https://github.com/zephyrproject-rtos/mbedtls this PR would just be a way of triggering CI with [the mbedtls PR](https://github.com/zephyrproject-rtos/mbedtls/pulls), not something which is meant to be merged.
But, after it is merged there, this line would need to be changed to point to the corresponding SHA, and then it could be merged here.

Reply:
of course

Reply:
Then I'd add `[DNM]` to a title, not just as a label...